qemu/tests/vsock_negative_test.py

```diff
@@ -0,0 +1,69 @@
+import random
+
+from avocado.utils import process
+
+from virttest import utils_misc
+from virttest import error_context
+
+from qemu.tests import vsock_test
+
+
+@error_context.context_aware
+def run(test, params, env):
+    """
+    Vsock negative test
+
+    S1):
+    1. Boot guest with vhost-vsock-pci device
+    2. Download and compile nc-vsock on both guest and host
+    3. Connect guest CID(on host) without listening port inside guest
+    S2):
+    1. Boot guest with vhost-vsock-pci device
+    2. Download and compile nc-vsock on both guest and host
```

Comment:
How about move the common step1 and step2 out of S1 and S2 ?   And it match the whole process.

Reply:
@yanan-fu Do you mean in the description? They are actually two different test cases, I want to keep it this way to make it clearer.

Reply:
@vivianQizhu  Yes, that is what i mean,  I just want to follow the whole process:  boot -> check S1 --> S2.  Not a big problem, It is up to you.

Reply:
@yanan-fu Thanks, the description is following our test case, and the code is following the description. Not should description follow code.